packages/generator-superset/README.md

```diff
@@ -0,0 +1,32 @@
+# generator-superset
+
+[![Version](https://img.shields.io/npm/v/@superset-ui/generator-superset.svg?style=flat)](https://img.shields.io/npm/v/@superset-ui/generator-superset.svg?style=flat-square)
+[![David (path)](https://img.shields.io/david/apache-superset/superset-ui.svg?path=packages%2Fgenerator-superset&style=flat-square)](https://david-dm.org/apache-superset/superset-ui?path=packages/generator-superset)
+
+> Scaffolder for Superset
+
+## Installation
+
+First, install [Yeoman](http://yeoman.io) and `generator-superset` using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
+```bash
+npm install -g yo
```

Comment:
1) is this recommended vs adding `yo` as a `dep` and getting it when you install this package? 
2) also wondering if we should suggest they just run `npm run bootstrap` in the root, and then `@superset-ui/generator-superset` + its deps will be installed?
3) if we do keep the separate installs, do we need them to be global packages?

Reply:
1/3) This is the recommended way as there should be single `yo` client. and `yo` looks for generator in the global scope. 
http://yeoman.io/learning/index.html

2) This `README` was written in a way that this packaged can be used independent from this repo. I envision this generator can be extended to have multiple subgenerator for scaffolding independent chart plugin repo, etc. It will be more convenient to install the generator package alone without cloning this monorepo only to `npm link` the generator.